app/controllers/sac_cas/groups/self_registration_controller.rb

```diff
@@ -5,15 +5,86 @@
 #  or later. See the COPYING file at the top-level directory or at
 #  https://github.com/hitobito/hitobito_sac_cas.
 
-module SacCas::Groups::SelfRegistrationController
-  extend ActiveSupport::Concern
+class SacCas::Groups::SelfRegistrationController < ApplicationController
```

Comment:
Lieber nicht den ganzen Controller ersetzen, sondern die Changes in den Core verschieben und dort Defaults wählen sodass die SelfRegistration im Core nicht gross abgeändert wird. Dann im SAC Wagon die effektiven SAC-spezifischen Steps einfügen.

Durch das Ersetzen hast du aus Versehen die `add_newsletter_label` Logik kaputt gemacht.